lib/rucio/core/subscription.py

```diff
@@ -48,28 +47,28 @@ def add_subscription(name, account, filter, replication_rules, comments, lifetim
     """
     Adds a new subscription which will be verified against every new added file and dataset
 
-    :param account: Account identifier
-    :type account:  String
-    :param name: Name of the subscription
-    :type:  String
-    :param filter: Dictionary of attributes by which the input data should be filtered
-                   **Example**: ``{'dsn': 'data11_hi*.express_express.*,data11_hi*physics_MinBiasOverlay*', 'account': 'tzero'}``
-    :type filter:  Dict
-    :param replication_rules: Replication rules to be set : Dictionary with keys copies, rse_expression, weight, rse_expression
-    :type replication_rules:  Dict
-    :param comments: Comments for the subscription
-    :type comments:  String
-    :param lifetime: Subscription's lifetime (days)
-    :type lifetime:  Integer or None
-    :param retroactive: Flag to know if the subscription should be applied on previous data
-    :type retroactive:  Boolean
-    :param dry_run: Just print the subscriptions actions without actually executing them (Useful if retroactive flag is set)
-    :type dry_run:  Boolean
-    :param priority: The priority of the subscription
-    :type priority: Integer
-    :param session: The database session in use.
-
-    :returns: The subscriptionid
+    :param account:            Account identifier
+    :type account:             String
+    :param name:               Name of the subscription
+    :type:                     String
```

Comment:
I am pretty sure it needs to be `:type name:` since you're already touchiing this :)